Source/JavaScriptCore/API/JSValue.h

```diff
@@ -339,7 +389,52 @@ NS_CLASS_AVAILABLE(10_9, 7_0)
 @result The NSDictionary containing the recursively converted contents of
  the converted JavaScript object.
 */
-- (NSDictionary *)toDictionary;
+- (NSDictionary * _Null_unspecified)toDictionary;
+
+/*!
+@method
+@abstract Convert a JavaScript BigInt to a NSString.
+@param radix The integer in the range 2 to 36 to be used as the base for representing the BigInt value.
+@result The NSString result of the conversion, a <code>nil</code> is returned if the JSValue is not a BigInt or the radix is not in the range 2 to 36.
+@discussion This is equivalent to calling the `BigInt::toString` from JavaScript with a radix, and equivalent to `toString` if the radix is 10.
+*/
+- (nullable NSString *)toBigIntString:(size_t)radix JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
```

Comment:
I wonder if this is too obscure to be worth adding. How often do people want then BigInt's stringified with radix != 10 that they can't just call a script with `BigInt.toString()`. Also, I think it would be `BigInt.toString` from JavaScript not `BigInt::toString`

Reply:
Yeah, let's remove them.